crates/sui-graphql-rpc/src/types/checkpoint.rs

```diff
@@ -60,12 +64,15 @@ pub(crate) struct Checkpoint {
     /// Representation of transaction data in the Indexer's Store. The indexer stores the
     /// transaction data and its effects together, in one table.
     pub stored: StoredCheckpoint,
+    /// Representation of the raw checkpoint data, including the summary and contents.
+    pub raw_checkpoint: Option<StoredRawCheckpoint>,
```

Comment:
No need to add this as an explicit field -- fetch it as part of the BCS field implementation.